library/CHART-TEMPLATE/tests/common_library_lowdatamode_test.yaml

```diff
@@ -0,0 +1,82 @@
+suite: test lowdatamode helper
+templates:
+  - templates/example-cm-lowdatamode.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+set:
+  licenseKey: test-license-key
+  cluster: test-cluster
+tests:
+  - it: Is false by default
+    asserts:
+      - equal:
+          path: data.lowDataMode-with-if
+          value: "disabled"
+      - equal:
+          path: data.lowDataMode-with-quote
+          value: ""
+      - equal:
+          path: data.lowDataMode-with-default
+          value: "false"
+
+  - it: Enable low data mode (globally)
+    set:
+      global:
+        lowDataMode: true
+    asserts:
+      - equal:
+          path: data.lowDataMode-with-if
+          value: "enabled"
+      - equal:
+          path: data.lowDataMode-with-quote
+          value: "true"
+      - equal:
+          path: data.lowDataMode-with-default
+          value: "true"
+
+  - it: Enable low data mode (locally)
+    set:
+      lowDataMode: true
+    asserts:
+      - equal:
+          path: data.lowDataMode-with-if
+          value: "enabled"
+      - equal:
+          path: data.lowDataMode-with-quote
+          value: "true"
+      - equal:
+          path: data.lowDataMode-with-default
+          value: "true"
+
+  - it: Overrides to false the global state locally
+    set:
+      global:
+        lowDataMode: true
+      lowDataMode: false
+    asserts:
+      - equal:
+          path: data.lowDataMode-with-if
+          value: "disabled"
+      - equal:
+          path: data.lowDataMode-with-quote
+          value: ""
+      - equal:
+          path: data.lowDataMode-with-default
+          value: "false"
+
+  - it: Overrides to true the global state locally
```

Comment:
it's funny that even having the test below , this test still make sense in helm 👍

Reply:
More than "a needed test" is the comment that @roobre put about having it only for `lowDataMode` and then I thought on putting more than one example.

It is stupid how we have to do falsiness-fu to overcome this but this should make things clear :)